src/styles/text/label.js

```diff
@@ -10,41 +12,81 @@ export default class Label {
             text,
             size,
             position: [],
-            bbox: [],
+            aabb: [],
             move_in_tile,
             keep_in_tile
         });
 
         this.id = Label.id++;
-
         this.buffer = this.buffer || 2; // TODO: make configurable
         this.buffer *= Geo.units_per_pixel;
+        this.keepMinDistance = true;
```

Comment:
Style in this codebase is `keep_min_distance` for variables, `camelCase()` for function/methods, (it's not my favorite but should stick to it for consistency).